docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -0,0 +1,143 @@
+| `summary`                             | A description of the pod built from this project (mandatory).                                                                                                                                                                 |
+| `homepage`                            | A link to the homepage of the pod built from this project (mandatory).                                                                                                                                                        |
+| `authors`                             | Specifies authors of the pod built from this project.                                                                                                                                                                         |
+| `podfile`                             | Configures the existing pod file.                                                                                                                                                                                             |
+| `noPodspec()`                         | Sets up plugin not to produce a podspec file for the cocoapods section.                                                                                                                                                       |
+| `useLibraries()`                      | Sets up `cocoapods-generate` to produce `xcodeproj` compatible with static libraries                                                                                                                                          |
+| `name`                                | The name of the pod built from this project. If not provided, the project name is used.                                                                                                                                       |
+| `license`                             | The license of the pod built from this project.                                                                                                                                                                               |
+| `framework`                           | The framework block configures the framework produced by the plugin.                                                                                                                                                          |
+| `source`                              | The location of the pod built from this project.                                                                                                                                                                              |
+| `extraSpecAttributes`                 | Configures other podspec attributes, like `libraries` or `vendored_frameworks`.                                                                                                                                               |
```

Comment:
podspec or pod spec?

Reply:
Yes, it should be Podspec, fixed throughout the documentation